charts/forseti-security/values.yaml

```diff
@@ -67,7 +67,7 @@ orchestrator:
   image: "gcr.io/forseti-containers/forseti"
 
   # orchestrator.imageTag is the tag for the orchestrator container image.
-  imageTag: "v2.21.0"
+  imageTag: "v2.22.0"
```

Comment:
Should these be 2.23? OR will this be changed when the release is out?

Reply:
Thanks for raising that question.  This will be updated during the release engineering process.